src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -109,6 +113,31 @@ protected virtual void Dispose(bool disposing)
             this.documentationBrowser.DpiChanged -= DocumentationBrowser_DpiChanged;
         }
 
+        async void InitializeAsync()
+        {
+            
+            string executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            var absoluteImagePath = Path.Combine(executingPath, FALLBACK_DOC_DIRECTORY_NAME);
+
+            //Initialize the CoreWebView2 component otherwise we can't navigate to a web page
+            await documentationBrowser.EnsureCoreWebView2Async();
+
+            //Due that the Web Browser(WebView2 - Chromium) security CORS is blocking the load of resources like images then we need to create a virtual folder in which the image are located.
+            this.documentationBrowser.CoreWebView2.SetVirtualHostNameToFolderMapping(VIRTUAL_FOLDER_MAPPING, FALLBACK_DOC_DIRECTORY_NAME, CoreWebView2HostResourceAccessKind.DenyCors);
```

Comment:
From your testing, how costly is this operation?

Reply:
@QilongTang not sure exactly what you mean by costly, but if you refer to the time that takes  WebView2 for loading the webpage and the image, at first glance I didn't notice any difference by using the full path "file:///C:/pathtoimage/image.png" against "http://virtualfolder/image.png", if you want I can check how we can measure the loading time and see the differences.

Reply:
I think you will need to consider how to this works for the other fallback docs folders as well, like the Revit/host one.
https://github.com/DynamoDS/Dynamo/blob/5e0337b06a2530facaf8f6b7fdc8355e8131c245/src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs#L33

Reply:
@mjkkirschner do you know if when Dynamo is running over Revit both paths (DynamoCoreDirectory, HostApplicationDirectory) could have values (see image below)? or just one and the other will be empty?
Because if only one has value then we should use an if ........ else if ...... structure, right? Just to not follow the same pattern.
![image](https://user-images.githubusercontent.com/61755417/184455347-1f57640e-ec7b-472b-af65-dfa4961b2bc7.png)

Reply:
yes, @RobertGlobant20 I think they will both have values, one folder will contain docs for core nodes, the other docs for the host.